Create 2023-04-28-content.md (#2247)
SumoLogic/sumologic-documentation: blog-cse/2023-04-28-content.md

---
title: April 28, 2023 - Content Release
hide_table_of_contents: true
image: https://help.sumologic.com/img/sumo-square.png
authors:
  - url: https://help.sumologic.com/release-notes-cse/rss.xml
    image_url: /img/release-notes/rss-orange.png
---

#### Rules
Updates several Azure based rules to account for modifications made to normalization mappers.

> * [New] MATCH-S00873 AWS EKS Cluster Configuration Updated; AWS EKS clusters contain various configuration options, including for which IP addresses can access the cluster API. Ensure that this change is authorized and expected.

Author: Acknowledged.

> * [New] MATCH-S00873 AWS EKS Cluster Configuration Updated; AWS EKS clusters contain various configuration options, including for which IP addresses can access the cluster API. Ensure that this change is authorized and expected.

Author: Commit pushed.

> * [New] MATCH-S00873 AWS EKS Cluster Configuration Updated; AWS EKS clusters contain various configuration options, including for which IP addresses can access the cluster API. Ensure that this change is authorized and expected.

Reviewer: ```suggestion
* [New] MATCH-S00873 AWS EKS Cluster Configuration Updated: AWS EKS clusters contain various configuration options, including for which IP addresses can access the cluster API. Ensure that this change is authorized and expected.
```

* [New] MATCH-S00872 AWS EKS Failed Curl Authentication Attempt; Failed instances of curl usage within a containerized environment should occur rarely, investigate the source IP address used to ensure that it is legitimate.

> * [New] MATCH-S00871 AWS EKS Pod Shared Object Modification or Creation; A Kubernetes pod was either created, updated or patched with a shared process namespace.

Reviewer: ```suggestion
* [New] MATCH-S00871 AWS EKS Pod Shared Object Modification or Creation: A Kubernetes pod was either created, updated, or patched with a shared process namespace.
```

> * [New] MATCH-S00870 AWS EKS Secrets Created; Kubernetes secrets may be created for legitimate purposes, ensure that this secrets created is from an IAM account that is expected to manage Kubernetes workloads on EKS.

Reviewer: ```suggestion
* [New] MATCH-S00870 AWS EKS Secrets Created: Kubernetes secrets may be created for legitimate purposes. Ensure that the secret created is from an IAM account that is expected to manage Kubernetes workloads on EKS.
```

Reviewer: I found another instance of "this secrets created is". Change both to "the secrete created is".

> * [New] MATCH-S00869 AWS EKS Secrets Deleted; Kubernetes secrets may be deleted for legitimate purposes, ensure that this secrets created is from an IAM account that is expected to manage Kubernetes workloads on EKS.

Reviewer: ```suggestion
* [New] MATCH-S00869 AWS EKS Secrets Deleted: Kubernetes secrets may be deleted for legitimate purposes. Ensure that this secrets created is from an IAM account that is expected to manage Kubernetes workloads on EKS.
```

* [New] FIRST-S00036 First Seen AWS EKS API Call via CloudTrail from User; The user {{user_username}} has performed an operation on an EKS cluster for the first time since the baseline period.

> * [New] FIRST-S00037 First Seen AWS EKS Admission Controller Created by IP Address; First Seen Admission Controllers (Submit a new MutatingWebhookConfiguration or ValidatingWebhookConfiguration object via the Kubernetes API, or update an existing one.)

Reviewer: ```suggestion
* [New] FIRST-S00037 First Seen AWS EKS Admission Controller Created by IP Address: First Seen Admission Controllers (submit a new MutatingWebhookConfiguration or ValidatingWebhookConfiguration object via the Kubernetes API, or update an existing one.)
```

> * [New] FIRST-S00035 First Seen AWS EKS Secrets Enumeration from IP Address; {{srcDevice_ip}} has enumerated secrets on an AWS EKS cluster for the first time since the baseline period.

Reviewer: ```suggestion
* [New] FIRST-S00035 First Seen AWS EKS Secrets Enumeration from IP Address: `srcDevice_ip` has enumerated secrets on an AWS EKS cluster for the first time since the baseline period.
```

> * [New] FIRST-S00034 First Seen Session Token Granted to User from New IP; An AWS Session token was issued for the first time since the baseline period to {{user_username}} using the IP address of {{srcDevice_ip}}

Reviewer: ```suggestion
* [New] FIRST-S00034 First Seen Session Token Granted to User from New IP: An AWS Session token was issued for the first time since the baseline period to `user_username` using the IP address of `srcDevice_ip`.
```

* [New] FIRST-S00033 First Seen Terminal-Attached Pod Deployed to EKS; A pod was deployed with an attached terminal (stdin=true,stdout=true,tty=true) for the first time since the baseline period.

> * [New] THRESHOLD-S00114 HTTP Response Error Spike to AWS EKS; HTTP web services provide response codes to client requests. The response code numbers in the 400s are used to indicate a client related error and response code numbers in the 500s represent server related errors. This rule looks for a AWS EKS cluster receiving a large frequency of web errors within a short period of time. It is unusual for a web client to cause this many errors in a short period of time. Common occurrences for this behavior is scanning/probing activity or scripted web clients which are now encountering errors due to a misconfiguration or recent change. This rule alerts when a host on the monitored network triggers the threshold.

Reviewer: ```suggestion
* [New] THRESHOLD-S00114 HTTP Response Error Spike to AWS EKS: HTTP web services provide response codes to client requests. The response code numbers in the 400s are used to indicate a client related error and response code numbers in the 500s represent server related errors. This rule looks for a AWS EKS cluster receiving a large frequency of web errors within a short period of time. It is unusual for a web client to cause this many errors in a short period of time. Common occurrences for this behavior is scanning/probing activity or scripted web clients which are now encountering errors due to a misconfiguration or recent change. This rule alerts when a host on the monitored network triggers the threshold.
```

> * [New] MATCH-S00868 New Binding Role Created on AWS EKS; A role binding grants a resource superuser or administrative access to a Kubernetes cluster. Ensure this action is expected and performed by known Kubernetes administrators.

Reviewer: ```suggestion
* [New] MATCH-S00868 New Binding Role Created on AWS EKS: A role binding grants a resource superuser or administrative access to a Kubernetes cluster. Ensure this action is expected and performed by known Kubernetes administrators.
```

> * [New] MATCH-S00867 New Cluster Admin Binding Role Created on AWS EKS; A cluster-admin role binding grants a resource superuser or administrative access to a Kubernetes cluster. Ensure this action is expected and performed by known Kubernetes administrators.

Reviewer: ```suggestion
* [New] MATCH-S00867 New Cluster Admin Binding Role Created on AWS EKS: A cluster-admin role binding grants a resource superuser or administrative access to a Kubernetes cluster. Ensure this action is expected and performed by known Kubernetes administrators.
```

* [New] MATCH-S00866 Privileged Pod Created on AWS EKS; Privileged containers have all capabilities of the host machine. These privileged containers may perform actions directly on the host that they are running on. Ensure that this event is expected and occurs from a user account or IP address that normally works with privileged containers within the cluster. Customers are encouraged to set up an exclusion list for spec.securitycontext.capabilities for pods that are frequently going to be managed with privileged escalation.
* [Updated] MATCH-S00864 Azure Firewall Rule Modified
* [Updated] MATCH-S00839 Azure Virtual Machine RunCommand Issued
* [Updated] FIRST-S00021 First Seen Azure Virtual Machine Run Command Issued by User
* [Updated] FIRST-S00032 First Seen Kubectl Command From User; Expanded record types considered to include Endpoint.
* [Updated] CHAIN-S00012 Potential Azure Persistence via Automation Accounts
* [Updated] MATCH-S00167 Recon Using Common Windows Commands; Narrowed rule criteria to Windows executables to prevent erroneous matches from *nix based systems

#### Log Mappers

> Adds support for Druva events via C2C. Modifies Darktrace mappers to include alternate field values and supports additional events. Modified CyberArk EPM mappers to include alternate field values. Adds support for DocuSign Monitor events via C2C. Adds support for Cisco Meraki events ingested via C2C. Includes updates to Microsoft Azure normalization to get more granular data for actions being performed. Adds additional mapping support for Jamf events. Corrects bug in severity mapping for Sysdig mappers. Expands mapping support for Workday logs ingested via C2C.

Reviewer: Are you sure you want this all as one paragraph? You might just want to add these pieces to lines below. For example:
"[New] Druva inSync - Catch All: Adds support for Druva events via C2C."

Author: We're trying out some new things and continuing to iterate on summaries versus in line comments. Would it make sense for trying it both ways and soliciting feedback?

Reviewer: @ptobia, I'd go one way or the other, but not both in the same set of release notes.

* [New] Administrator Audit Trail
* [New] Administrator Logon
* [New] Darktrace Parser - Anomalous Connection
* [New] Darktrace Parser - Brute Force Attempt
* [New] DocuSign Monitor - Alert
* [New] DocuSign Monitor - Catch All
* [New] Druva inSync - Catch All
* [New] Jamf Audit User - Authentication
* [New] Jamf Audit User - Endpoint
* [New] Jamf Audit User - Network
* [New] Workday - Sign On
* [Updated] Azure Administrative logs

> * [Updated] Cisco Meraki IDS Alert - C2C; Corrects typo in mapper for some IP/port fields

Reviewer: ```suggestion
* [Updated] Cisco Meraki IDS Alert - C2C: Corrects typo in mapper for some IP/port fields
```
